apps/storefront/docs/patterns/spacings/spacings.mdx

```diff
@@ -4,3 +4,35 @@ metaTitle: Spacings
 mode: draft
 route: /patterns/spacings/
 ---
+
+**Using the correct spacing throughout you design will provide consistency to your product**
+
+## Spacing Reminder
+
+Spacing is used to apply margin and padding across all platforms. The EDS has a predefined set of spacers. These predefined sizes will ensure consistency across interfaces. The spacers XX Small and X Small are only to be used inside of components, while the spacers Small up to XXX Large can be used both within and between components.
+
+Always make sure to follow the accessibility guidelines by having 8px of spacing between each clickbound.
+
+## Guidelines
+
+Using a grid as a starting point for your component implementation will help you arrange your content into structured propositions. You also have to keep in mind that EDS is based on an 8 px ratio. This means that you always have to keep a minimum of 8px beetween objects and elements 
+
+### Spacing according to buttons
+
+#### Spacing beetween buttons:
+Button pairs such as “Continue” and “Cancel” are used with 8px spacing between each button. Buttons with no connection are used with at least 16 px spacing between them.
+
+#### Buttons below components
+Buttons placed below a form or table with a connection to the form or table should have a spacing of 8px. 
+
+#### Buttons next to othe components
+Buttons placed next to an input field should have a spacing of 8px.
+
+### Spacing according to input fields
+
+Input fields follow the grid both horisontally and vertically (Check grid alignment). Between multiple input fields there should be a spacing of 16px, and a maximum of two fields per row.
+
+### Spacing according to cards
+
+Cards follow the grid both horisontally and vertically (Check grid alignment). Between multiple cards there should be a spacing of at least 16px.
```

Comment:
Make the c lowercase in (Check grid alignment) since that is the writing style to only capitalize the first letter of a sentence.

Reply:
Updated according to review